### PR TITLE
Fix protect objectives

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

order of operations moment

## Why It's Good For The Game

bugfix

## Changelog
:cl:
fix: Protect objectives now properly greentext
/:cl:

### CODE REVIEW

File: code/game/gamemodes/objectives/basic/protect.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	if(human_check)
 		brain_target = target?.current.getorganslot(ORGAN_SLOT_BRAIN)
 	//Protect will always suceed when someone suicides
-	return ..() || !target || considered_alive(target, enforce_human = human_check) || (human_check == TRUE && brain_target)? brain_target.suicided : FALSE
```
Comment (Absolucy) [inline code/game/gamemodes/objectives/basic/protect.dm:17]: Yeah, uh, the `?` took the _entire left side_

**Comment by itsmeow** [comment]
I thought I already fixed this in the Valentine's overhaul

**Comment by itsmeow** [comment]
image
@PowerfulBacon I swear to god did you break it

**Comment by itsmeow** [comment]
bacon why
image
image

**Comment by itsmeow** [comment]
Just re implement my changes

**Comment by PowerfulBacon** [comment]
Must've been eaten by a merge conflict resolution

**Comment by Absolucy** [comment]
changes rem-impelmtned

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**